docs/rest-server/API.md

```diff
@@ -456,8 +456,19 @@ Status: 200
     appLaunchedTime: "applicationLaunchedTimestamp",
     appCompletedTime: "applicationCompletedTimestamp",
     appExitCode: applicationExitCode,
-    appExitDiagnostics: "applicationExitDiagnostics"
-    appExitType: "applicationExitType"
+    appExitDiagnostics: "applicationExitDiagnostics",
+    appExitType: "applicationExitType",
+    // please check https://github.com/Microsoft/pai/blob/errorspec/src/job-exit-spec/config/job-exit-spec.md for more information
```

Comment:
>https://github.com/Microsoft/pai/blob/errorspec/src/job-exit-spec/config/job-exit-spec.md [](start = 20, length = 89)

This branch will be deleted.
Please use relative link, like:
[runtime-exit-spec.md](../../../docs/rest-server/runtime-exit-spec.md)

Reply:
But using master branch still may different with the current branch of API.md. Could you please move the comment out of the json code block, so you can use markdown syntax?

---
In reply to: [276979745](https://github.com/Microsoft/pai/pull/2550#discussion_r276979745) [](ancestors = 276979745)

Reply:
You expand the object in the json.
Then explain them out of the code block by link.
Is that ok?

---
In reply to: [277195557](https://github.com/Microsoft/pai/pull/2550#discussion_r277195557) [](ancestors = 277195557,276979745)